### PR TITLE
Adding new landing page CTA.

### LINKED ISSUE #2282: As a new funder, there isn't a way to get to the funder landing page at /onboard/funder.
### User Story

As a new funder, there isn't a way to get to the funder landing page at /onboard/funder. The landing page to begin the funder FTUX is https://gitcoin.co/onboard/funder/, however, there is currently no way for an unregistered or registered user to land on `onboard/funder` except by directly typing the URL into the address bar. 

### Why Is this Needed

It should be made more clear where a new funder "begins" their journey on the homepage.

### Current Behavior

Currently, there isn't a way to get to the funder landing page at https://gitcoin.co/onboard/funder/.
In order for a new funder to connect their Github, they have to click "Log In" at the top right hand corner, and connect their Github information.

### Solution

New funders, regardless of where they enter, should always be deposited into the funder FTUX funnel. There should be an intuitive start location on the homepage that links directly to `/onboard/funder`.

### Definition of Done

Link https://gitcoin.co/onboard/funder/ to an intuitive CTA on the homepage that makes it clear where new funders go to sign up and enter the onboarding flow. This ticket is related to https://github.com/gitcoinco/web/issues/2283, which is the contributor CTA.

### PR DESCRIPTION
**Pull request by Anish-Agnihotri**
##### Description

Introduces two new CTA sections in the green banner on the Gitcoin landing page. These link to the funder/contributor guides, and the respective get-started funnels. Everything is 100% responsive and works across all devices, although a review on the JQuery in `landing_page.js` would be much appreciated.

##### Checklist

- [X] linter status: 100% pass
- [X] Glitch (see video at 0:42) fixed.
- [X] changes don't break existing behavior
- [X] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)
-  [X] Design an intuitive CTA on the homepage that makes it clear where new contributors go to sign up and subsequently enter the onboarding flow.
- [X] for smaller screens and mobile, build the CTAs as viewed on larger screens into a rotating carousel so it doesn't suck up the mobile screen.

##### Affected core subsystem(s)

UI

##### Testing

Tested across two devices and various resolutions as usual. Feel free to test on your local machines as well. 

##### Refers/Fixes

Fixes #2282, #2283.

### CODE REVIEW

**Comment by Anish-Agnihotri** [comment]
Correct! Apologies haven’t gotten the time to drop by since I’m knocking out Gitcoin bot atm. Just need to make a quick fix for an existing bug, and add auto-rotation to Carousel on smaller displays.

Will remove WIP tomorrow. 

Thanks!

Anish

**Comment by octavioamu** [comment]
@Anish-Agnihotri ping me when ready to review, for now looks good. Thanks!

**Comment by Anish-Agnihotri** [comment]
@willsputra 

Following our conversation on the issue regarding swiping on mobile, I've first increased the size of the left/right buttons: 

large2
large1

I tried to get touch up and running through a simple fix: 
```
myElement.addEventListener("touchstart", startTouch, false);
myElement.addEventListener("touchmove", moveTouch, false);
 
var initialX = null;
 
function startTouch(e) {
  initialX = e.touches[0].clientX;
};
 
function moveTouch(e) {
  if (initialX === null) {
    return;
  }
 
  var currentX = e.touches[0].clientX;
 
  var diffX = initialX - currentX;
 
    if (diffX > 0) {
      // swiped left
      console.log("swiped left");
    } else {
      // swiped right
      console.log("swiped right");
    }  
 
  initialX = null;
   
  e.preventDefault();
};
```

But, this is a super super messy solution. I shot a message in Slack about possibly importing an external library to handle the carousel altogether, simply because all these features are already defined, and well tested in such libraries. My recommendation would be [Owl-Carousel](https://owlcarousel2.github.io/OwlCarousel2/) which is 44kB minified and has all the features we need + more, and it has CSS2 fallback support in case of ancient browsers support. 

Of course though, if this is against Gitcoin's policies, I'd be glad to clean up this code, get some reviews, and figure out a better way to handle carousel :)!

Thanks,

Anish

**Comment by thelostone-mc** [comment]
@Anish-Agnihotri I still think reusing what @octavioamu had laid out might make things easier.
It uses [scroll-type ](https://css-tricks.com/practical-css-scroll-snapping/) and you can check out the code in [process_bounty](https://github.com/gitcoinco/web/blob/master/app/dashboard/templates/process_bounty.html#L98)

**Comment by Anish-Agnihotri** [comment]
Sounds good. I'll review and integrate it tonight, and have PR ready for push today.

**Comment by thelostone-mc** [comment]
@Anish-Agnihotri good for review?

**Comment by Anish-Agnihotri** [comment]
Going to open this one up for review, so that I can get it merged. Feel free to review, and I'll knock out any issues.

File: app/assets/v2/js/landing_page.js [hunk 1]
```diff
@@ -69,3 +69,46 @@ $(document).ready(function() {
...
+  var i;
+
+  var x = document.getElementsByClassName('split-cta');
+
+  if (n > x.length) {
+    slideIndex = 1;
+  }
+  if (n < 1) {
+    slideIndex = x.length;
+  }
```
Comment (thelostone-mc) [inline app/assets/v2/js/landing_page.js]: change it to `if-else`

**Comment by Anish-Agnihotri** [comment]
Thanks for review! Fixing and pushing within 24 hrs. Just having some trouble fixing my docker instance atm.

**Comment by frankchen07** [comment]
Also like to add a check that each of the CTAs go to the correct onboarding flow for contributors and funders - almost there!

**Comment by Anish-Agnihotri** [comment]
Finally managed to fix my docker last night (was having trouble and got some help in #community-help). Will be knock this out tonight.

**Comment by Anish-Agnihotri** [comment]
This issue hit the backburner real hard. I'll knock it out tonight and make a PR in 24hrs closing it for good.

**Comment by thelostone-mc** [comment]
C

**Pull request closed without merging**